rockets/ws/messageHandler.cpp

```diff
@@ -46,10 +46,10 @@ void MessageHandler::handleMessage(Connection& connection, const char* data,
     if (format == Format::text)
     {
         if (callbackText)
-            response = callbackText(std::move(_buffer));
+            response = callbackText({std::move(_buffer), reinterpret_cast<uintptr_t>(&connection) });
```

Comment:
define a client_id variable to make the intent clear and remove the 3 casts? Also, I think static_cast is preferable to reinterpret_cast if possible (for objects at least).